src/server/game/AuctionHouseBot/AuctionHouseBot.cpp

```diff
@@ -146,7 +146,6 @@ void AuctionBotConfig::GetConfigFromFile()
     SetConfigMax(CONFIG_AHBOT_NEUTRAL_ITEM_AMOUNT_RATIO, "AuctionHouseBot.Neutral.Items.Amount.Ratio", 100, 10000);
 
     SetAHBotIncludes(sConfigMgr->GetStringDefault("AuctionHouseBot.forceIncludeItems", ""));
-    SetAHBotExcludes(sConfigMgr->GetStringDefault("AuctionHouseBot.forceExcludeItems", ""));
```

Comment:
> Anyway, as far as losing data on a db update, you would lose that anyway because those entries are entirely user defined so the user would need to maintain a separate SQL file for their AHBot stuff, along with the other AHBot options in the CONF file.

Which wasn't lost before with tdb updates since those entries are in the conf file.

What about adding a deprecation notice to config and core?